### PR TITLE
R4 - Update MedicationRequest Overview

### PR DESCRIPTION
**Pull request by PorosnicuCornel**
Description
----
Updates for the FHIR resource: MedicationRequest.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

**Screenshot(s) before the changes:**
----

image

**Screenshot(s) after the changes:**
----

image

image

### CODE REVIEW

**Review by MariaPislaru1 — APPROVED** [review]

**Review by lstefan4 — APPROVED** [review]

**Comment by davischristi** [comment]
Can you add the screenshot of the listed profile?

**Comment by PorosnicuCornel** [comment]
> 

image

**Review by davischristi — CHANGES_REQUESTED** [review]
Also remove the link to the jira from the description.

**Comment by PorosnicuCornel** [comment]
> Also remove the link to the jira from the description.

done

File: content/millennium/r4/clinical/medications/medication-request.md [hunk 1]
```diff
@@ -25,25 +29,42 @@ The following fields are returned if valued:
...
+      * [durationUnit](https://hl7.org/fhir/r4/datatypes-definitions.html#Timing.repeat.durationUnit){:target="_blank"} \| [code](https://hl7.org/fhir/r4/datatypes.html#code){:target="_blank"}
+      * [bounds](https://hl7.org/fhir/r4/datatypes-definitions.html#Timing.repeat.bounds_x_){:target="_blank"}
+        * boundsPeriod \| [Period](https://hl7.org/fhir/r4/datatypes.html#Period){:target="_blank"}
   * [As Needed](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.asNeeded_x_){:target="_blank"}
   * [Body Site](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.site ){:target="_blank"}
   * [Route](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.route){:target="_blank"}
   * [Dose and Rate](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.doseAndRate){:target="_blank"}
+    * [doseAndRate.dose\[x\]](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.doseAndRate.dose_x_){:target="_blank"}
+    * [doseAndRate.rate\[x\]](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.doseAndRate.rate_x_){:target="_blank"}
 * [Dispensing Details](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest){:target="_blank"}
```
[the remark this answers is not in the document]
Comment (PorosnicuCornel) [reply content/millennium/r4/clinical/medications/medication-request.md:63]: image
done

File: content/millennium/r4/clinical/medications/medication-request.md [hunk 2]
```diff
@@ -25,25 +25,42 @@ The following fields are returned if valued:
 * [Priority](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.priority){:target="_blank"}
 * [Reported Boolean](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.reported_x_){:target="_blank"}
 * [Medication](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.medication_x_){:target="_blank"}
+  * [CodeableConcept](https://hl7.org/fhir/r4/datatypes.html#CodeableConcept){:target="_blank"}
```
[the remark this answers is not in the document]
Comment (PorosnicuCornel) [reply content/millennium/r4/clinical/medications/medication-request.md:32]: image

done

**Review by davischristi — APPROVED** [review]

**Comment by ebbp123** [comment]
Looks like some spaces were added in line 8 which broke the table of contents for the resource: 
Screen Shot 2022-09-28 at 3 11 56 PM

**Comment by ebbp123** [comment]
PR to Fix the TOC: https://github.com/cerner/fhir.cerner.com/pull/905

**Pull request merged**